JUnitTestsForStreamAPI/src/WordCounter.java

```diff
@@ -0,0 +1,23 @@
+import java.util.Arrays;
+
+public class WordCounter {
+    /*
+    Написать функцию, которая принимает предложение, а возвращает
+количество слов, начинающихся на заданную букву.
+     */
+
+    public static long countWords(String st, String letter){
+        if(st == null || letter == null){
+            System.out.println("Input arguments are wrong!");
+            return 0;
```

Comment:
In such cases (when the input is not valid) you may consider throwing IllegalArgumentException instead of returning 0:

`throw new IllegalArgumentException("Input arguments are wrong!)`;

We are about to lear this.